### PR TITLE
Highlight section on hover

### LINKED ISSUE #138: Highlight the section on hover

### PR DESCRIPTION
**Pull request by Elkrival**
* We highlight the data of the section we hover over

Screen Shot 2022-08-19 at 4 46 15 PM

### CODE REVIEW

File: views/components/Graphs/BarGraph.jsx [hunk 1]
```diff
@@ -67,6 +67,36 @@ const BarGraph = ({ graph }) => {
...
+                eventHandlers: {
+                  onMouseOver: () => {
+                    return [
+                      {
+                        mutation: (props) => {
+                          return {
+                            style: Object.assign({}, props.style, {
+                              stroke: colors.black,
+                              strokeWidth: 1,
+                            }),
```
Comment (mikestone14) [inline views/components/Graphs/BarGraph.jsx]:
```suggestion
                            style: {
                              ...props.style,
                              stroke: colors.black,
                              strokeWidth: 1,
                            },
```

File: views/components/Graphs/BarGraph.jsx [hunk 2]
```diff
@@ -67,6 +67,36 @@ const BarGraph = ({ graph }) => {
...
+                  },
+                  onMouseOut: () => {
+                    return [
+                      {
+                        mutation: () => {
+                          return null
+                        },
+                      },
+                    ]
+                  },
```
Comment (mikestone14) [inline views/components/Graphs/BarGraph.jsx:98]: is this part needed?

**Review by mikestone14 — APPROVED** [review]
looks good just a couple minor things.

File: views/components/Graphs/BarGraph.jsx (hunk 2 above)
Comment (Elkrival) [reply views/components/Graphs/BarGraph.jsx:98]: Without this the bar stays highlighted.

**Review by mikestone14 — APPROVED** [review]

**Pull request merged**